build.gradle

```diff
@@ -126,7 +126,7 @@ dependencies {
     // Plugin dependencies
     compileOnly group: 'org.opensearch', name:'opensearch-ml-client', version: "${opensearch_build}"
     implementation fileTree(dir: jsJarDirectory, include: ["opensearch-job-scheduler-${opensearch_build}.jar"])
-    implementation fileTree(dir: adJarDirectory, include: ["opensearch-time-series-analytics-${opensearch_build}.jar"])
+    implementation fileTree(dir: adJarDirectory, include: ["opensearch-anomaly-detection-${opensearch_build}.jar"])
```

Comment:
AD team plans to change the jar name from `opensearch-anomaly-detection` to `opensearch-time-series-analytics`, but main and 2.x and release branch are not consistent, during 2.14 release, this inconsistency caused skills failed to build. @kaituo can you confirm which name will be using in main and 2.x in 2.15 release?

Reply:
Based on latest main, @kaituo changed back to `opensearch-anomaly-detection` in [#1210](https://github.com/opensearch-project/anomaly-detection/pull/1210/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7L206)